services/common/src/main/java/org/collectionspace/services/common/ServiceMain.java

```diff
@@ -974,7 +1011,8 @@ private HashSet<String> createNuxeoDatabases() throws Exception {
 							JDBCTools.createNewDatabaseUser(JDBCTools.CSADMIN_DATASOURCE_NAME, repositoryName, cspaceInstanceId, dbType, readerUser, readerPW);
 						}
 						// Create the database
-						createDatabaseWithRights(dbType, dbName, nuxeoUser, nuxeoPW, readerUser, readerPW);
+						createDatabaseWithRights(dbType, dbName, nuxeoUser, nuxeoPW, readerUser);
+						createUtilsSchemaWithRights(dbType, nuxeoUser, repositoryName, cspaceInstanceId);
```

Comment:
The call to create the utils schema. Could also be removed unless it really is necessary.

Reply:
I think this is fine to leave in. We can probably find a use for it.